src/Miki/Modules/Accounts/AccountsModule.cs

```diff
@@ -1140,39 +1141,46 @@ await builder.ToEmbed()
                 return;
             }
 
-            int streak = 0;
-            string redisKey = $"user:{e.GetAuthor().Id}:daily";
+            ICacheClient cache = e.GetService<ICacheClient>();
+            string redisKey = $"user:{user.Id}:daily";
 
-            var cache = e.GetService<ICacheClient>();
+            if (await cache.ExistsAsync(redisKey).ConfigureAwait(false))
```

Comment:
Avoid using cache operations directly in commands. Rather, let the service handle it